templates/textinput.jsx

```diff
@@ -30,6 +30,7 @@ export default function TextInput (props) {
         ])}
         aria-labelledby={ariaQuestion ? null : (displayTitle || body || instruction) && `${_id}-header`}
         aria-label={ariaQuestion || null}
+        role='group'
```

Comment:
Re group vs region see https://github.com/adaptlearning/adapt-contrib-confidenceSlider/pull/113#pullrequestreview-2177068396

Reply:
Confidence slider updated to group for consistency which resolves this query.